scripts/track-dependency-status.sh

```diff
@@ -0,0 +1,48 @@
+# Get total count of python dependencies and outdated python dependencies
+
+total_python_deps="$(pip list| wc -l)"
+outdated_python_deps="$(pip list --outdated| wc -l)"
```

Comment:
Nice, I had no idea this existed :) Much nicer than a custom bash script!

Reply:
Yeah. After finding the command for yarn, I was looking into pip's documentation for something like that and I found it.